tests/lib_catch2.h

```diff
@@ -24,6 +24,8 @@
 
 #if __has_include(<Catch2/single_include/catch2/catch.hpp>)
 #include <Catch2/single_include/catch2/catch.hpp>
```

Comment:
Including headers this way looks like a hack. Isn't it possible to tell Visual Studio to add the various subprojects to the include path for tests?

Reply:
It already is set up to do that; the top-level Catch2 is the subproject.

I could have easily added the deeper path, of course. Doing it this way was mostly out expediency - there was never any danger of it being incorrect as it was a git submodule tied to a specific commit.